script/Common/00_00.txt

```diff
@@ -1098,48 +1102,50 @@ Going to that field in the afternoon became part of my daily routine.
 [sha:679f0655b670ed3bd944520bcf94c08bf0f8e0d1]{
 -- Page 77, Offset 317.
 -- 女の人は青子と呼ぶとおこる。自分の名前が嫌いなんだそうだ。
-The woman would get upset if I called her Aoko.
+She gets upset if I call her Aoko.
  She doesn't seem to like her own name.
 }
 [sha:cb9be31be1cfa806d8bd173f7920b2d36617fd60]{
 -- Page 77, Offset 318.
 -- なやんだあげく、なんとなく偉そうな人だから『先生』と呼ぶことにした。
-After much pondering, I decided to call her 'Sensei' because she seemed like an upstanding person.
+// not sure if past tense should be preserved here
+After much pondering, I decided to call her 'Sensei' because she seemed like such an authoritative person.
 }
 [sha:2f8f600b75d75c6f4d01e082c8e0eea25faf2c0f]{
 -- Page 78, Offset 319.
 -- 先生はどんな話でも真面目に聞いてくれて、僕の悩みを一言で解いてくれる。
-Sensei listened to me seriously no matter what I said, and solved my problems with just a few words.
+Sensei listens to me seriously no matter what I say, and solves my problems with just a few words.
 }
 [sha:c964e04d6c46f0bcb6514d46bea2913b6345b267]{
 -- Page 79, Offset 320.
 -- 事故のあと、人と話すことが苦手になっていた僕は、先生のおかげで少しずつ元の自分に戻っていけた。
-After the accident, I had grown uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
+// not sure about tense here
+After the accident, I had grown uncomfortable talking to people, but thanks to her, I am able to gradually return to my normal self.
```

Comment:
Might drop first comma

Reply:
I feel like I'm gradually returning to my normal self.